integration-test/src/commonTest/kotlin/sample/JsonTest.kt

```diff
@@ -212,22 +209,6 @@ class JsonTest {
         )
     }
 
-    @Test
-    fun testZoo() {
```

Comment:
This test is intentionally copypasted from the main library to check that behavior does not change in userland environment